docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - "8080:8080"
+    environment:
+    - ENV=local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image:
+      mysql:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
```

Comment:
The `build` section is unnecessary if you are specifying an image for the `mysql` service. You should either build the image using a Dockerfile or pull it from a registry, but not both.